src/components/MessagesList/MessagesList.vue

```diff
@@ -259,13 +262,17 @@ export default {
 		},
 		chatIdentifier: {
 			immediate: true,
-			handler() {
+			handler(newValue, oldValue) {
+				if (oldValue) {
+					this.$store.dispatch('cancelLookForNewMessages', { requestId: oldValue })
+				}
 				this.handleStartGettingMessagesPreconditions()
 			},
 		},
 	},
 
 	mounted() {
+		this.viewId = uniqueId('messagesList')
```

Comment:
This breaks uploading files, creating folder and deleting either in the files app